proxywasm/hostcall.go

```diff
@@ -589,70 +590,82 @@ func CallForeignFunction(funcName string, param []byte) (ret []byte, err error)
 	}
 }
 
-// LogTracef emits a message as a log with Trace log level.
+// LogLevel represents the current log level.
+var LogLevel = log.LevelTrace
```

Comment:
Ok so I see this is the only thing you want to have right? what's the advantage to have this in our SDK vs you implementing this in your own program? 

This is particularly confusing as we already have "log level" in host ABI level and you are trying to introduce LogLevel in Go level here.